app/javascript/components/downloads/Downloads.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import axios from 'axios';
+import { PropTypes } from 'prop-types';
+import { Button } from 'react-bootstrap';
+import reportError from '../util/ReportError';
+import { ToastContainer, toast } from 'react-toastify';
+
+class Downloads extends React.Component {
```

Comment:
From a UI perspective, this adds an extra step the user needs to perform in order to download each file which could be time consuming for exports with multiple files. Even though these are just small UI changes, should we run them by the PO team? especially for language like "This download has been marked for removal", I'm not sure what users would think as it sounds like something more for developers to see.

Not sure if this is outside the scope of this ticket, but I was wondering if we could just provide a single button/link that would just download all the files from an export (so users wouldn't have to click on each of them individually) which can be done individually or in a zip file

Reply:
We definitely want to combine the exports back into one file (no more splitting up, all types should be tabs in one file rather than separate files).

Reply:
I removed the separate files and now full exports are all in a single workbook with multiple worksheets